src/netgroup.cpp

```diff
@@ -109,3 +110,23 @@ uint32_t NetGroupManager::GetMappedAS(const CNetAddr& address) const
     uint32_t mapped_as = Interpret(m_asmap, ip_bits);
     return mapped_as;
 }
+
+void NetGroupManager::ASMapHealthCheck(const std::vector<CNetAddr> clearnet_addrs) const {
+    std::set<uint32_t> clearnet_asns{};
+    int unmapped_count{0};
+
+    for (auto addr : clearnet_addrs) {
```

Comment:
I think you need to declare `addr` as `const` (`tidy`CI test is failing due to that). 
```suggestion
    for (const auto &addr : clearnet_addrs) {
```